infra/build/functions/build_lib.py

```diff
@@ -327,19 +349,34 @@ def get_git_clone_step(repo_url='https://github.com/google/oss-fuzz.git',
   return clone_step
 
 
+def _armify_image_name(image_name):
+  """Returns an ARM-specific name for |image_name|."""
+  return image_name + '-arm'
+
+
 def get_docker_build_step(image_names,
                           directory,
                           buildkit_cache_image=None,
-                          src_root='oss-fuzz'):
+                          src_root='oss-fuzz',
+                          is_arm=False):
```

Comment:
Rather than pass these `is_arm` arguments everywhere, could we instead just pass the target architecture?

We could add the known target architecture somewhere in a constant (i.e. x86_64), and do a check below and take the `buildx` path if they don't match. This may also make later refactoring easier if GCB starts supporting arm64.

Reply:
If GCB offers arm64 we should just yank out buildx stuff. It won't be useful for any purpose. I changed to passing around the architecture.